### DISCUSSION TITLE
[Feature] Language options on the immich.app website

### DISCUSSION THREAD

**Discussion by Deedikjupijn** [immich-app/immich#9400] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a language button like the dark/light mode options on the menu bar in the website.

One of the goals of this project is to bring privacy to everyone and I would assume that it is a crucial step to add multiple language support to all parts of this project to make it as accessible as possible. 

This would go hand in hand with translations already existing on the android app making sure everyone is able to get privacy even without the knowledge of the English language.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by martabal** [comment]
Duplicate of #1688
